src/Image.cc

```diff
@@ -1223,6 +1230,78 @@ Image::loadSVG(FILE *stream) {
 
 #endif /* HAVE_RSVG */
 
+/*
+ * Load BMP from buffer.
+ */
+
+cairo_status_t Image::loadBMPFromBuffer(uint8_t *buf, unsigned len){
+  BMPParser::Parser parser;
+
+  // Reversed ARGB32 with pre-multiplied alpha
+  uint8_t pixFmt[5] = {2, 1, 0, 3, 1};
+  parser.parse(buf, len, pixFmt);
+
+  if (parser.getStatus() != BMPParser::Status::OK) {
+    errorInfo.reset();
+    errorInfo.cerrno = 0;
```

Comment:
Minor: don't need to set this to 0; `reset()` does that.

Reply:
fixed